admin/base/filters.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+from django.utils.safestring import mark_safe
+from django.utils import simplejson
+
+register = template.Library()
+
+@register.filter('jsonify')
```

Comment:
You don't technically need to pass jsonify to filter. It'll use the function name by default.